PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -171,3 +171,11 @@
 _phase2_timing_extraCommands = ["keep *_offlineSlimmedPrimaryVertices4D_*_*"]
 from Configuration.Eras.Modifier_phase2_timing_cff import phase2_timing
 phase2_timing.toModify(MicroEventContentMC, outputCommands = MicroEventContentMC.outputCommands + _phase2_timing_extraCommands)
+
+_pp_on_AA_extraCommands = [
+    'keep booledmValueMap_*MuonID_*_*',
+]
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+(pp_on_AA_2018 | pp_on_PbPb_run3).toModify(MicroEventContent, outputCommands = MicroEventContent.outputCommands + _pp_on_AA_extraCommands)
+(pp_on_AA_2018 | pp_on_PbPb_run3).toModify(MicroEventContentMC, outputCommands = MicroEventContentMC.outputCommands + _pp_on_AA_extraCommands + ['keep *_heavyIon_*_*'])
```

Comment:
- what's heavyIons ? this seems unrelated to this PR
- the common modifications of MicroEventContent should be done before MicroEventContentMC is made out of it on L136
- as commented in the review, it seems like these valuMaps should not be saved (they can not be used as saved)

Reply:
Removed the + ['keep *_heavyIon_*_*'] part as this was introduced by mistake, and moved the common modifier before the MicroEventContentMC is made out.